Makefile

```diff
@@ -353,34 +358,35 @@ ifeq ($(UNAME_S),Linux)
-		--set apigatewayURL=localhost:8080 \
-		--set consoleURL=localhost:3000 \
-		--set console.serverApiGatewayBaseUrl=localhost:8080
-	@sleep 1
+		--set triton.nvidiaVisibleDevices=${TRITON_NVIDIA_VISIBLE_DEVICES} \
+		--set apigatewayURL=http://localhost:8080 \
+		--set consoleURL=http://localhost:3000 \
+		--set console.serverApiGatewayBaseUrl=http://localhost:8080
+	@sleep 10
 	@export CONTROLLER_POD_NAME=$$(kubectl get pods --namespace vdp -l "app.kubernetes.io/component=controller,app.kubernetes.io/instance=vdp" -o jsonpath="{.items[0].metadata.name}") && \
 		kubectl wait --for=condition=Ready pod $$CONTROLLER_POD_NAME -n vdp --timeout=300s || true
 	@export APIGATEWAY_POD_NAME=$$(kubectl get pods --namespace vdp -l "app.kubernetes.io/component=api-gateway,app.kubernetes.io/instance=vdp" -o jsonpath="{.items[0].metadata.name}") && \
 		export APIGATEWAY_CONTAINER_PORT=$$(kubectl get pod --namespace vdp $$APIGATEWAY_POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}") && \
-	kubectl --namespace vdp port-forward $$APIGATEWAY_POD_NAME 8080:$${APIGATEWAY_CONTAINER_PORT} > /dev/null 2>&1 &
+		kubectl --namespace vdp port-forward $$APIGATEWAY_POD_NAME 8080:$${APIGATEWAY_CONTAINER_PORT} > /dev/null 2>&1 &
 	@export CONSOLE_POD_NAME=$$(kubectl get pods --namespace vdp -l "app.kubernetes.io/component=console,app.kubernetes.io/instance=vdp" -o jsonpath="{.items[0].metadata.name}") && \
 		export CONSOLE_CONTAINER_PORT=$$(kubectl get pod --namespace vdp $$CONSOLE_POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}") && \
 		kubectl --namespace vdp port-forward $$CONSOLE_POD_NAME 3000:$${CONSOLE_CONTAINER_PORT} > /dev/null 2>&1 &
 	@docker run -it --rm --network host --name backend-helm-integration-test-release instill/vdp-compose:release /bin/bash -c " \
-		cd pipeline-backend && make integration-test API_GATEWAY_HOST=host.docker.internal API_GATEWAY_PORT=8080 && cd ~- && \
-		cd connector-backend && make integration-test API_GATEWAY_HOST=host.docker.internal API_GATEWAY_PORT=8080 && cd ~- && \
-		cd model-backend && make integration-test API_GATEWAY_HOST=host.docker.internal API_GATEWAY_PORT=8080 && cd ~- && \
-		cd mgmt-backend && make integration-test API_GATEWAY_HOST=host.docker.internal API_GATEWAY_PORT=8080 && cd ~- \
+		cd pipeline-backend && make integration-test API_GATEWAY_HOST=localhost API_GATEWAY_PORT=8080 && cd ~- && \
+		cd connector-backend && make integration-test API_GATEWAY_HOST=localhost API_GATEWAY_PORT=8080 && cd ~- && \
+		cd model-backend && make integration-test API_GATEWAY_HOST=localhost API_GATEWAY_PORT=8080 && cd ~- && \
+		cd mgmt-backend && make integration-test API_GATEWAY_HOST=localhost API_GATEWAY_PORT=8080 && cd ~- \
 		"
 	@docker run -it --rm \
-		-e NEXT_PUBLIC_CONSOLE_BASE_URL=http://console:3000 \
-		-e NEXT_PUBLIC_API_GATEWAY_BASE_URL=http://api-gateway:8080 \
+		-e NEXT_PUBLIC_CONSOLE_BASE_URL=http://localhost:3000 \
+		-e NEXT_PUBLIC_API_GATEWAY_BASE_URL=http://localhost:8080 \
```

Comment:
```
		-e NEXT_PUBLIC_CONSOLE_BASE_URL=http://console:3000 \
		-e NEXT_PUBLIC_API_GATEWAY_BASE_URL=http://api-gateway:8080 \
```
These two lines are actually fine under `--network host` on Linux. But I think just use `localhost` to be more specific.